.travis.yml

```diff
@@ -69,6 +69,14 @@ matrix:
       env: TOXENV=py34
     - python: "3.5"
       env: TOXENV=py35
+    - os: osx
+      osx_image: xcode7.3
+      sudo: required
+      language: generic
+      env: TOXENV=cover
+      before_install: "travis_retry brew install python augeas dialog"
```

Comment:
I'm aware, the problem is that `brew update` broke earlier builds for some reason. I was waiting for Travis builds to run to be able to determine whether it broke because of `brew update` or not.

I'm not ignoring @bmw, don't worry. Thanks a lot for the note! :)